setup.cfg

```diff
@@ -29,6 +29,7 @@ install_requires =
     astropy-healpix
     cached-property
     deprecation
+    hera-cli-utils>=0.1.0
     numpy>=1.18
     pyuvdata
```

Comment:
We should pin `pyuvdata` at least to the version that require the new frequency array shape, maybe even the more recent version, I am not sure. This is implicitly enforced from `pyuvsim` requirement, I think, but being explicit is better.

Reply:
Pinned to 2.2.12